4.x/webpack/demo/src/css/main.css

```diff
@@ -0,0 +1,2 @@
+@import "~arcgis-js-api/assets/esri/themes/light/main.css";

```

Comment:
You used the dark css theme in the dojo example, maybe make this dark for consistency?